paint.sh

```diff
@@ -15,5 +15,5 @@ do
 		export GIT_AUTHOR_DATE="$d 12:$m:$s"
 		git commit --date="$d 12:$m:$s" -m "$i on $d" --no-gpg-sign --allow-empty
 	done
-done < dates.txt
+done < ${0%/*}/dates.txt
```

Comment:
As paths with spaces may split this argument into multiple segments be sure to quote properly.
This also fails if `$0` does not contain a directory name, because paint.sh is on the `$PATH`. Use `dirname` instead.

Reply:
done, thanks!